### PR TITLE
Progress further in json config implementation

### PR DESCRIPTION
**Pull request by igor725**

### CODE REVIEW

File: tools/config/config.cpp [hunk 1]
```diff
@@ -2,23 +2,120 @@
 #include "config.h"
 #undef __APICALL_EXTERN
 
+#include "logging.h"
+#include <fstream>
+
+LOG_DEFINE_MODULE(Config);
```
Comment (igor725) [inline tools/config/config.cpp:8]: It throws an undefined symbol error on every logging call there.

Comment (SysRay) [reply tools/config/config.cpp:9]: // No logging here (logging depends on config )
// Can be put together with logging if needed

replace the traces with printf and remove logging. At above as reminder.

Comment (igor725) [reply tools/config/config.cpp:9]: Oooh, right. I forgot about it, lol...

File: modules/libkernel/types.h [hunk 2]
```diff
@@ -91,3 +91,9 @@ struct rusage_t {
   uint32_t         ru_nvcsw    = 0;
   uint32_t         ru_nivcsw   = 0;
 };
+
+struct timesec {
```
Comment (SysRay) [inline modules/libkernel/types.h:95]: Name it SceTimesce pls. So, it's not getting mixed up with any later internal timesec when porting.

**Pull request merged**